pkg/filesystem/fs/resolver.go

```diff
@@ -12,24 +12,23 @@ import (
 	"net/http"
 	"time"
 
-	"github.com/containerd/containerd/log"
 	"github.com/containerd/containerd/reference/docker"
 	"github.com/containerd/nydus-snapshotter/pkg/auth"
-	"github.com/containerd/nydus-snapshotter/pkg/utils/registry"
+	"github.com/containerd/nydus-snapshotter/pkg/utils/transport"
 	"github.com/google/go-containerregistry/pkg/name"
 	retryablehttp "github.com/hashicorp/go-retryablehttp"
 	"github.com/pkg/errors"
 )
 
-const HTTPClientTimeOut = time.Second * 30
+const HTTPClientTimeOut = time.Second * 60
```

Comment:
Is this used?

Reply:
used, default HTTP retry client's retry timeout

Reply:
```
func newRetryHTTPClient(tr http.RoundTripper) *retryablehttp.Client {
	retryClient := retryablehttp.NewClient()
	retryClient.HTTPClient.Transport = tr
	retryClient.HTTPClient.Timeout = HTTPClientTimeOut
	retryClient.Logger = nil
	return retryClient
}
```